src/main/java/io/vlingo/actors/Stage.java

```diff
@@ -739,6 +723,38 @@ private Definition redefinitionWithMailboxName(final Definition definition, fina
     return redefinition;
   }
 
+  <T> Actor rawLookupOrStart(Definition definition, Address address) {
+    Actor actor = directory.actorOf(address);
+    if (actor != null) {
+      return actor;
+    }
+    try {
+      return createRawActor(definition, definition.parentOr(world.defaultParent()), address, null, definition.supervisor(), world.defaultLogger());
+    } catch (Directory.ActorAddressAlreadyRegistered ignored) {
+      return rawLookupOrStart(definition, address);
+    }
+  }
+
+  <T> T lookupOrStart(Class<T> protocol, Definition definition, Address address) {
+    return actorAs(actorLookupOrStart(definition, address), protocol);
+  }
+
+  <T> Actor actorLookupOrStart(Definition definition, Address address) {
+    Actor actor = directory.actorOf(address);
+    if (actor != null) {
+      return actor;
+    }
+    else {
+      try {
+        actorFor(Startable.class, definition, address);
```

Comment:
@VaughnVernon, perhaps `NoProtocol` is more appropriate than `Startable`? Could you please provide some insight about the former?

Reply:
Every `Actor` is `Startable`, it's just a matter of the client requesting that protocol as a proxy. I think you'd have a problem using `NoProtocol` because Actor does not implement that one and it not being implemented would cause failure in the `ActorFactory` or during proxy creation since there is a check for that.